packages/web/src/pages/sign-on-page/AudiusValues.tsx

```diff
@@ -34,6 +36,7 @@ const AudiusValue = (props: AudiusValueProps) => {
         strength={isDesktop ? 'default' : 'weak'}
         color='staticWhite'
         shadow='emphasis'
+        css={dynamic && { fontSize: '1.9vw' }}
```

Comment:
Using `vw` for font size seems strange to me, was there a special reason for it?

Reply:
Also, feels kind of unnecessary to be that dynamic imo. What if we just swapped to mobile size here instead?

Reply:
yeah on further review, i agree, going to revert back to mobile styles